pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1921,7 +1934,59 @@ impl RemoteTimelineClient {
             }
 
             let upload_result: anyhow::Result<()> = match &task.op {
-                UploadOp::UploadLayer(ref layer, ref layer_metadata) => {
+                UploadOp::UploadLayer(ref layer, ref layer_metadata, last_op) => {
```

Comment:
`last_op` is a confusing name here: in the code path that inserts this operation we call this `mode`?

Reply:
ahh, a commit is lost, I remembered I renamed this

Reply:
did a force push and the missing commit shows up in GitHub https://github.com/neondatabase/neon/pull/9844/commits/78c02f32459e4cfbeaebd0f00568889941119a0d